kyuubi-server/src/main/scala/org/apache/kyuubi/sql/plan/command/DescribeEngine.scala

```diff
@@ -35,13 +35,19 @@ import org.apache.kyuubi.sql.schema.{Column, Row, Schema}
 case class DescribeEngine() extends RunnableCommand {
 
   override def run(kyuubiSession: KyuubiSession): Unit = {
-    val rows = Seq(kyuubiSession).map { session =>
-      lazy val client = session.asInstanceOf[KyuubiSessionImpl].client
-      val values = new ListBuffer[String]()
-      values += client.engineId.getOrElse("")
-      values += client.engineName.getOrElse("")
-      values += client.engineUrl.getOrElse("")
-      Row(values.toList)
+    val rows = Seq(kyuubiSession.asInstanceOf[KyuubiSessionImpl]).flatMap { session =>
+      lazy val client = session.client
+      session.listZkEngineNodes.map { nodeInfo =>
+        val values = new ListBuffer[String]()
+        values += client.engineId.getOrElse("")
+        values += client.engineName.getOrElse("")
+        values += client.engineUrl.getOrElse("")
+        values += nodeInfo.namespace
+        values += s"${nodeInfo.host}:${nodeInfo.port}"
+        values += nodeInfo.version.getOrElse("")
+        values += nodeInfo.attributes.filter(_._1.contains("memory")).mkString(";")
```

Comment:
Is it necessary to filter Engine zk attributes? @turboFei

Reply:
not needed